src/test/java/org/terracotta/offheapstore/util/PhysicalMemoryTest.java

```diff
@@ -32,34 +33,44 @@
  * @author cdennis
  */
 public class PhysicalMemoryTest {
-  
+
   @Test
   public void testMemoryInvariants() {
     Assert.assertThat(PhysicalMemory.freePhysicalMemory(), anyOf(nullValue(Long.class), lessThanOrEqualTo(PhysicalMemory.totalPhysicalMemory())));
     Assert.assertThat(PhysicalMemory.freeSwapSpace(), anyOf(nullValue(Long.class), lessThanOrEqualTo(PhysicalMemory.totalSwapSpace())));
     Assert.assertThat(PhysicalMemory.ourCommittedVirtualMemory(), anyOf(nullValue(Long.class), greaterThanOrEqualTo(0L)));
   }
-  
+
   @Test
   public void testBehaviorWithSecurityManager() {
-    final Thread testThread = Thread.currentThread();
+    Thread testThread = Thread.currentThread();
     System.setSecurityManager(new SecurityManager() {
-
-      @Override
-      public void checkMemberAccess(Class<?> clazz, int which) {
-        if (Thread.currentThread() == testThread && OperatingSystemMXBean.class.isAssignableFrom(clazz)) {
-          throw new SecurityException();
-        }
-      }
-
       @Override
       public void checkPackageAccess(String pkg) {
         if (Thread.currentThread() == testThread && pkg.startsWith("com.sun.")) {
           throw new SecurityException();
         }
       }
 
+      @Override
       public void checkPermission(Permission perm) {
```

Comment:
Was the intention here to avoid overriding the deprecated method?  I ask because otherwise the test seems to work unmodified.  In any case I think that the following will work okay (and is simpler):
```java
@Override
public void checkPermission(Permission perm) {
  if (Thread.currentThread() == testThread && CHECK_MEMBER_ACCESS.equals(perm)) {
    throw new SecurityException();
  }
}
```

Reply:
Also `SecurityConstants` is private API... which seems annoying, but should we therefore use our own constant here?

Reply:
But we do no need to check if it's called by `OperatingSystemMXBean`? That looked like the whole point of the test.

Reply:
Oups for the contant. Fixed.